src/main/java/org/folio/linked/data/service/resource/ResourceMarcAuthorityService.java

```diff
@@ -8,4 +8,6 @@ public interface ResourceMarcAuthorityService {
   Long saveMarcResource(Resource modelResource);
 
   org.folio.linked.data.model.entity.Resource fetchResourceOrCreateFromSrsRecord(Identifiable identifiable);
+
+  Boolean isMarcValidForTargetAssignment(String marc, AssignAuthorityTarget target);
```

Comment:
'Valid/validation' makes me think it's like a validation part of some request handling, like creation or update.
But it's not, right? It's separate request only for the check.

Reply:
agree, changed to ```isMarcCompatibleWithTarget(String marc, AssignAuthorityTarget target)```